tools/create-package-components.js

```diff
@@ -49,20 +48,14 @@ module.exports = task('create-package-components', async () => {
   // Create directory
   await fs.ensureDir(packageDir)
 
+  // eslint-disable-next-line no-console
   console.info('Package name will be:', packageName)
```

Comment:
Is the console.log intended to be here?

Reply:
Yeah! This script is a developer tool for scaffolding out a new component + directory structure, so it isn't publicly available. Just for CLI feedback 👍